examples/LoRaWAN/Starter/Starter.ino

```diff
@@ -0,0 +1,41 @@
+#include <Arduino.h>
```

Comment:
@HeadBoffin Would it possible to keep the `LoRaWAN_*` naming? The CI uses the sketch name to filter based on regex.